src/Controller/TrickController.php

```diff
@@ -68,7 +72,10 @@ public function add(Request $request, SluggerInterface $slugger, EntityManagerIn
      * @return Response
      */
     public function edit(TrickRepository $trickRepository, Trick $trick, Request $request, SluggerInterface $slugger,
-                         EntityManagerInterface $em, SessionInterface $session){
+                         EntityManagerInterface $em, SessionInterface $session): Response
+    {
+
+        $this->denyAccessUnlessGranted('MANAGE', $trick);
```

Comment:
`$this->denyAccessUnlessGranted('ROLE_USER');`

Reply:
modifié